packages/emulation/libretro-opera/package.mk

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2021-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="libretro-opera"
+PKG_VERSION="5781f68e39e4af41aede43467964948af30a2547"
```

Comment:
kodi's game addon uses 539dc447b536ba52eb3485f11d9bf1c06e30cf75 https://github.com/kodi-game/game.libretro.opera/blob/master/depends/common/opera/opera.txt

please use that version so we are in sync